src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/vcf/VcfFuncotationFactory.java

```diff
@@ -87,11 +88,20 @@
     //==================================================================================================================
     // Constructors:
 
+    /**
+     * Use this only for testing,
+     */
+    @VisibleForTesting
     public VcfFuncotationFactory(final String name, final String version, final Path sourceFilePath) {
```

Comment:
Do you really need yet another visible-for-testing constructor here? Can't you just use the one below in tests?

Reply:
No, I can use the other one.  Fixed.